.github/workflows/cloud-integration-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Cloud Integration Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      job_to_run:
+        description: 'Job to run'
```

Comment:
because this is a literal, please clearly state all the possible options a user can input

Reply:
And the behaviour of each input